command.php

```diff
@@ -22,7 +22,9 @@ class WooCommerce_Blocks_Testing_Environment extends WP_CLI_Command {
 	 * ## OPTIONS
 	 *
 	 * [--blocks[=<version>]]
-	 * : The desired WooCommerce Blocks version to install. When selecting 'true', the most recent version gets installed.
+	 * : The desired WooCommerce Blocks version to install.
+	 * Specify a version number to install a specific release. Leave empty to install the latest version.
+	 * Specify a link to the zip version to install from a remote source.
```

Comment:
If you'd accept my suggestion before, what about adjusting this text to highlight the various parameter? And what are your thoughts on updating the `EXAMPLES` so that it reflects the different parameter and arguments?

Reply:
Hey, great suggestion, I have improved a bit the `--blocks` argument documentation and examples.
Let me know if what I did is more or less what you had in mind or if you would prefer it some other way. 😄